Solutions/Forescout/Data Connectors/Forescout_syslog.json

```diff
@@ -0,0 +1,118 @@
+{
+    "id": "Forescout",
+    "title": "Forescout",
+    "publisher": "Forescout",
+    "descriptionMarkdown": "The [Forescout](https://www.forescout.com/) data connector provides the capability to ingest [Forescout events](https://docs.forescout.com/bundle/syslog-3-6-1-h/page/syslog-3-6-1-h.How-to-Work-with-the-Syslog-Plugin.html) into Azure Sentinel. Refer to [Forescout documentation](https://docs.forescout.com/bundle/syslog-msg-3-6-tn/page/syslog-msg-3-6-tn.About-Syslog-Messages-in-Forescout.html) for more information.",
+    "additionalRequirementBanner": "This data connector depends on a parser based on a Kusto Function to work as expected [**ForescoutEvent**](https://aka.ms/sentinel-forescout-parser) which is deployed with the Azure Sentinel Solution.",
+    "graphQueries": [{
+        "metricName": "Total data received",
+        "legend": "Forescout",
+        "baseQuery": "ForescoutEvent"
+    }],
+    "sampleQueries": [{
+        "description": "Top 10 Sources",
+        "query": "ForescoutEvent\n | summarize count() by tostring(DvcHostname)\n | top 10 by count_"
```

Comment:
Can you please check and fix this query.
![image](https://user-images.githubusercontent.com/87809732/130061061-dba80a81-260a-47d6-ab7e-e947821fded0.png)

Reply:
fixed